test/vanilla/src/main/java/fixtures/bodyformdata/Formdatas.java

```diff
@@ -71,39 +71,43 @@
     /**
      * Upload file.
      *
+     * @param contentLength The content length.
      * @param fileContent File to upload.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
      * @throws ErrorException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      * @return the Flowable&lt;ByteBuffer&gt; object if successful.
      */
-    Flowable<ByteBuffer> uploadFileViaBody(@NonNull Flowable<ByteBuffer> fileContent);
+    Flowable<ByteBuffer> uploadFileViaBody(@NonNull long contentLength, @NonNull Flowable<ByteBuffer> fileContent);
```

Comment:
```java
@NonNull long contentLength
```
I guess this is a bug in the generator. I think it's pretty clear that long will never be null.

Reply:
I guess it's not needed, but I also don't foresee it causing users any pain. Are primitives in general having this annotation applied?

Reply:
It looks like they are. I'm surprised, because I thought I had handled this case.